res/res_agi.c

```diff
@@ -73,6 +73,11 @@
 
 /*** DOCUMENTATION
 	<agi name="answer" language="en_US">
+		<since>
+			<version>0.1.0</version>
+			<version>1.6.1.0</version>
+			<version>1.8.0</version>
+		</since>
```

Comment:
Shouldn't this just be 0.1.0? This applies to most of this file.

Reply:
I'm not sure what's right or wrong any more. :)

Reply:
At least I indented the versions though.